### PR TITLE
R 4.0 with m2w64-pcre2

### PR DESCRIPTION
**Pull request by xhochy**

### CODE REVIEW

**Comment by xhochy** [comment]
wow, this works on CI.

Will clean this up and make a PR to staged-recipes.

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

File: recipe/bld.bat [hunk 1]
```diff
@@ -6,6 +6,16 @@
 @rem that it doesn't block the terminal, but we also can't get the return
 @rem value for the conda build tests.
 
+curl -C - -o "mingw-w64-x86_64-pcre2-10.34-1-any.pkg.tar.xz" -SLO http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-pcre2-10.34-1-any.pkg.tar.xz
```
Comment (bgruening) [inline recipe/bld.bat:9]: Aye! That is cool!

Comment (xhochy) [reply recipe/bld.bat:9]: That shouldn't be needed anymore. There is now a conda package for that.

**Comment by dpryan79** [comment]
Nicely done @xhochy!

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Comment by xhochy** [comment]
This is ready for merge. The Travis failure is spurious, there is no Travis config anymore in this PR so the build shouldn't have been triggered at all.

**Comment by dpryan79** [comment]
Definite :+1: from me, great work!

**Pull request merged**